docs/source/federation-spec.md

```diff
@@ -435,3 +435,38 @@ interface Product {
   hidden: String @inaccessible
 }
 ```
+
+### `@tag`
+
+```graphql
+directive @tag(from: String!) on FIELD_DEFINITION | INTERFACE | OBJECT | UNION | ARGUMENT_DEFINITION | SCALAR | ENUM | ENUM_VALUE | INPUT_OBJECT | INPUT_FIELD_DEFINITION
```

Comment:
**blocker** This should be marked as `repeatable` and the input param is called `name`, to align with what's in code / the spec https://specs.apollo.dev/tag/v0.2/#@tag

```suggestion
directive @tag(name: String!) repeatable on FIELD_DEFINITION | INTERFACE | OBJECT | UNION | ARGUMENT_DEFINITION | SCALAR | ENUM | ENUM_VALUE | INPUT_OBJECT | INPUT_FIELD_DEFINITION
```